app/__init__.py

```diff
@@ -227,4 +230,4 @@ def get_minimized_asset(filename):
 
 
 def restrict_content_length(application):
-    application.config['MAX_CONTENT_LENGTH'] = settings.EQ_MAX_HTTP_POST_CONTENT_LENGTH
+    application.config['MAX_CONTENT_LENGTH'] = application.config['EQ_MAX_HTTP_POST_CONTENT_LENGTH']
```

Comment:
Would it make sent to make `EQ_MAX_HTTP_POST_CONTENT_LENGTH` `MAX_CONTENT_LENGTH` so that this doesn't need to be set?